### PR TITLE
Cyprus (House of Representatives): refresh data

### PR DESCRIPTION
**Pull request by everypoliticianbot**
Automated data refresh for Cyprus - House of Representatives

#### Output

```
Add memberships from sources/morph/openpatata.csv
Merging with sources/morph/wikidata.csv
Adding GenderBalance results from sources/gender-balance/results.csv
  ⚥ data for 0; 0 added

Creating 2 term files

Top identifiers:
  92 x openpatata
  64 x parliament_cy
  34 x wikidata
  3 x europarlmep
  3 x freebase

Creating names.csv
Creating unstable/positions.csv
  Unknown position (x4): Q24307961 Member of House of Representatives (Cyprus) for the constituency of Famagusta — e.g. Q15950881
  Unknown position (x1): Q24307971 Member of House of Representatives (Cyprus) for the constituency of Keryneia — e.g. Q12882425
  Unknown position (x3): Q24307958 Member of House of Representatives (Cyprus) for the constituency of Larnaca — e.g. Q16329183
  Unknown position (x6): Q24307956 Member of House of Representatives (Cyprus) for the constituency of Limassol — e.g. Q351655
  Unknown position (x12): Q24307968 Member of House of Representatives (Cyprus) for the constituency of Nicosia — e.g. Q12876791
  Unknown position (x2): Q24307964 Member of House of Representatives (Cyprus) for the constituency of Paphos — e.g. Q12873769
  Unknown position (x2): Q19358487 President of House of Representatives (Cyprus) — e.g. Q8053482
Persons matched to Wikidata: 34 ✓ | 58 ✘
  No wikidata: Erotokritou Christiana (ea9b0162-bf07-4bdb-a1e6-c2c673655372)
  No wikidata: Sampson Sotiris (11021b19-d92c-4d72-b702-cf8fbe957626)
  No wikidata: Konstantinou Kostas (0986e289-d83f-4be6-bc1b-14295726c8c1)
  No wikidata: Efstathiou Kostis (4962ae16-0481-4495-b734-b15f0419c863)
  No wikidata: Koulla Onoufrios (43b6ffe3-83c7-4eb5-83a0-9e2611b8f85c)
  No wikidata: Papagiannis Linos (ed0d0585-739e-478b-aa04-63437c00bb2d)
  No wikidata: Myrianthous Ilias (663cd9aa-9ccc-4764-9e51-cfa823d50493)
  No wikidata: Sizopoulos Marinos (265f3b63-cb2a-41be-91f8-fcc6120ca4f9)
  No wikidata: Themistokleous Andreas (28cc782d-fd7d-4f4d-b217-45be9bed4598)
  No wikidata: Varnava Giorgos (e58e3fc9-d354-4374-9d7d-a95ae503a715)
Parties matched to Wikidata: 5 ✓ | 6 ✘
  No wikidata: Solidarity Movement (allilengyi)
  No wikidata: Citizens’ Alliance (sypol)
  No wikidata: Independent (_IND)
  No wikidata: Cyprus Greens – Citizens Cooperation (oikologoi-sp)
  No wikidata: National Popular Front (elam)

```

### CODE REVIEW

File: data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv [hunk 1]
```diff
@@ -91,3 +91,4 @@ stavrou-eleni,1300a45f-e959-4c03-86ea-0e617453d75a
 stefanou-stefanos,5f7ae1a1-3120-46bb-8b12-3db943c74571
 theologou-anna,f66b51ea-891c-4fbf-adf0-6d78a8ad29d5
 theopemptou-charalampos,bf608732-89a6-467a-802a-bd301d2e72eb
+georgiou-k-giorgos,d8a3a81b-678d-46b7-ba6e-e39e68dd2ecc
```
Comment (wfdd) [inline data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv]: @tmtmtmtm: I think we'd also need to get rid of `georgiou-giorgos` above - he's been duplicated in the CSV, though the new ID is completely absent from the JSON file, oddly.

Comment (tmtmtmtm) [inline data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv]: Having multiple mapping entries should be fine — that means no matter which we see, it'll end up doing the right thing. Do you just mean there's duplication in _this_ CSV file, or in one of the output files?

Comment (wfdd) [inline data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv]: Sorry, I mistook the source CSV for the term CSV. What appears to be happening is `georgiou-giorgos` is processed first and new data does not overwrite existing data, and, and, and...

We can either flush the database on morph.io and scrape the openpatata dump anew or replace `georgiou-giorgos` with `georgiou-k-giorgos` in the `ids` CSV - or do both.

Comment (tmtmtmtm) [inline data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv]: Ah, if it's an issue with the Morph data, then removing the 'georgiou-giorgos' line here won't help — all that will happen then is that when it processes that entry it'll give it another new UUID. If you're going to be changing IDs for people semi-regularly, we should probably make the scraper cope with that in some way.

Comment (wfdd) [inline data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv]: Seems to be all sorted now.  I suppose we could drop the data table before each run, though it's probably a better idea I don't change any other IDs.

**Pull request merged**